### PR TITLE
Convert DGXie to Helm chart

### PR DESCRIPTION
**Pull request by dholt**
Needed to make DGXie configuration possible without modifying the kubernetes manifest

### CODE REVIEW

**Comment by dholt** [comment]
@nvhans do you mind testing this out when you get a chance?

File: services/dgxie/values.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+netPrvDhcpStart: 192.168.1.100
+netPrvDhcpEnd: 192.168.1.199
+netPrvDhcpLease: 7200
+httpPort: 13370
+
+# Deployment config
+replicaCount: 1
+
+image:
+  repository: dgxie
```
Comment (ryanolson) [inline services/dgxie/values.yaml]: should this be `registry.local/deepops/dgxie`?

Comment (dholt) [reply services/dgxie/values.yaml]: currently you build the dgxie container locally and copy it over to the mgmt host and load it in via docker, so it's not coming from the registry. it probably would be a better idea to bootstrap the registry and put the container there, but this config correct for now

Comment (ryanolson) [reply services/dgxie/values.yaml]: 👍

**Pull request merged**